migrations/etl/1_Mine_profile_detail_location.sql

```diff
@@ -71,7 +71,8 @@ BEGIN
         longitude      = mms.mmsmin.lon_dec,
         major_mine_ind = transform_major_mine_ind(mms.mmsmin.min_lnk),
         mine_region    = transform_mine_region(mms.mmsmin.reg_cd)    ,
-        mine_type      = transform_mine_type_code(mms.mmsmin.mine_typ)
+        mine_type      = transform_mine_type_code(mms.mmsmin.mine_typ),
+        deleted_ind    = CASE WHEN lower(mms.mmsmin.mine_nm) LIKE '%delete%' OR lower(mms.mmsmin.mine_nm) LIKE '%deleted%' OR lower(mms.mmsmin.mine_nm) LIKE '%reuse%' THEN TRUE ELSE FALSE
```

Comment:
It looks like this is missing and `END`

Also, the `CASE` itself should be redundant here. This should give you the same thing:
```sql
deleted_ind = LOWER(mms.mmsmin.mine_nm) LIKE '%delete%' OR LOWER(mms.mmsmin.mine_nm) LIKE '%deleted%' OR LOWER(mms.mmsmin.mine_nm) LIKE '%reuse%'
```

Reply:
since the case is in a select statement or the part of a statement you don't want the END.If I add the END it won't run and this currently works on my dev branch.

Reply:
I also tried swapping out the case for this logical statement and postgres wouldn't run it.

Reply:
Hmm... that doesn't seem right to me, since I can run this query just fine:
```
mds=> select LOWER(mine_nm) LIKE '%delete%' OR LOWER(mine_nm) LIKE '%deleted%' OR LOWER(mine_nm) LIKE '%reuse%' AS deleted_ind from mms.mmsmin LIMIT 10;
 deleted_ind
-------------
 f
 f
 f
 f
 f
 f
 f
 f
 f
 f
(10 rows)
```

Reply:
I added this back in to the first case but I think I had added it for both the first time. It worked this time around.